spec/views/form_drafts/edit_spec.rb

```diff
@@ -0,0 +1,107 @@
+require 'rails_helper'
+include RSpecHtmlMatchers
+
+describe 'form_drafts/edit.haml' do
+  before :each do
+    @draft = create :form_draft
+    @top_field = create :field, form_draft: @draft
+    @field = create :field, form_draft: @draft
+    @bottom_field = create :field, form_draft: @draft
+  end
+  it 'contains the name of the form draft' do
+    render
+    expect(rendered).to have_tag 'h1', with: { class: 'title' }
+  end
+  it 'has a form to edit the form draft' do
+    render
+    action_path = form_draft_path @draft
+    expect(rendered).to have_form action_path, :post
```

Comment:
I'd also like to see a test about the attributes of the form draft which can be edited (name).